doc/arch/adr-014-track-attribute-changes-per-basepath.md

```diff
@@ -0,0 +1,87 @@
+# Track attribute changes through time for b    asepaths
```

Comment:
There's too many spaces in basepath `b    asepaths`

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/924/commits/1f9f9618978f6a8546991d544b404d8a52ab93e7